org.sigxcpu.Livi.json

```diff
@@ -98,8 +99,8 @@
                 {
                     "type" : "git",
                     "url" : "https://gitlab.gnome.org/guidog/livi",
-                    "tag" : "v0.0.6",
-                    "commit" : "8c73da4959db7e5291a64fda108ff1e26c0eac35"
+                    "tage" : "0.1.0",
```

Comment:
Whoops, I made a typo and changed `tag` to `tage` - but shouldn't matter as the commit is still correct. But should be fixed on the next update :P